examples/pytorch/stgcn_wave/model.py

```diff
@@ -0,0 +1,100 @@
+import math
+import torch
+import torch.nn as nn
+import torch.nn.init as init
+import torch.nn.functional as F
+from dgl.nn.pytorch import GraphConv
+from dgl.nn.pytorch.conv import ChebConv
+
+class temporal_conv_layer(nn.Module):
```

Comment:
Rename this (and all class names in this file) to CamelCase instead of snake_case.  In general all Python class names are better in CamelCase and functions/methods/variables/etc. are better in snake_case.